### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by danielnuk**
This PR is:

- [- ] Adding a new domain
- [ ] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [- ] I used the uncompressed size of the site
- [- ] I have included a link to the GTMetrix report
- [- ] The domain is in the correct alphabetical order
- [- ] This site is not an ultra lightweight site
- [- ] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [- ] Check to confirm

```
- domain: danieln.uk
  url: https://danieln.uk
  size: 106
  last_checked: 2023-09-04
```

GTMetrix Report: https://gtmetrix.com/reports/danieln.uk/JzLwJpyC/

### CODE REVIEW

File: _data/sites.yml [hunk 1]
```diff
@@ -818,6 +818,11 @@
   size: 18.8
   last_checked: 2022-04-26
 
+  - domain: danieln.uk
```
Comment (garritfra) [inline _data/sites.yml:821]: This indentation seems off. Do you mind fixing it?

**Review by garritfra — CHANGES_REQUESTED** [review]

**Review by danielnuk — COMMENTED** [review]
Whoops. Updated :)

**Comment by garritfra** [comment]
Thanks! :)

**Review by garritfra — APPROVED** [review]

**Comment by garritfra** [comment]
Any reason you closed this PR?

**Pull request closed without merging**